### ISSUE TITLE
Scheme reports/forms section: "License"

### ISSUE THREAD

**Issue by robgaston** [archesproject/arches-lingo#121]
# preamble

to build out the sections of the scheme forms/reports we will need to:

1. define mappings between form/report fields and the Lingo models
2. implement backend serializers/APIs
3. implement front end UI

# mapping issues

![scheme license:rights](https://github.com/user-attachments/assets/0a2cef9c-1a59-4a55-acef-30a20dfa7086)

It's not clear where in the model these data are stored, but I suspect it is in the "Rights" nodegroup & its children.  Is this correct? the fields don't seem to align in any meaningful way, so I assume i've either not identified the right nodegroup or the mockups are not complete relative to the model.

@Habennin @Philbox @dwuthrich we should discuss & resolve the above questions as they will be blockers to development here. thanks!

**Comment by Philbox** [comment]
@robgaston I'm not sure what Version means in the UI here but there is a version nodegroup which is intended to record the version of a published scheme. I'm not sure if we ever nailed down what exactly constitutes a version. 
I think that the licence does/should refer to the Rights group and will be the Rights Statement
The licence is intended to specify what rights affect the use of the vocabulary. For example we make ours available under the open government licence https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/

@Habennin will know more about the modelling

**Comment by Habennin** [comment]
I agree with @Philbox , rights version is not something foreseen in the modelling as a separate field. Rather the idea was that you have a controlled list of rights types like CC BY 4 and if there is a version maybe there is a child CCBY 4, Version 4 or something like that. No need in my opinion for an extra field to put the version of the right type in.

What is foreseen is putting who the right belongs to and also a statement field to put further information or to just write out what your rights information is if you don't use a stanard license. So basically what is foreseen in the model is the whole node group.

From the mockup the closest field is right_type. 

<img width="513" alt="Screenshot 2024-11-07 at 4 59 41 PM" src="https://github.com/user-attachments/assets/2b09fe29-f8f8-4d71-8627-0c0b7ef21da2">

**Comment by robgaston** [comment]
@dwuthrich what did we have in mind when mocking this up?  does `right_type` seem to fit this?  wondering if we should be exposing other fields related to this in Lingo as the nodegoups are much richer than that one field

**Comment by robgaston** [comment]
The license section should probably be renamed "Scheme Rights", which will include the type and holder drop downs.   This tile is cardinality 1 and will be the parent of the rights statements. We can then add another section for "Scheme Rights Statements" which can manage the fields that make up the right_statement (cardinality n).  This section will be similar to the "Scheme Notes" section.

@chrabyrd we should be unblocked on these (now 2) sections of the Scheme forms/reports.

Note: we will need to mockup data for the "right_type" and "right_holder" dropdowns.  @Habennin can you please take a look at the `right_type` controlled list as you look at the CLM generally and ensure that we populate it with the right entries?  thanks!
